server/docs/getting-started.md

```diff
@@ -2,106 +2,251 @@
+```
+
+
+----------
 
-1. Instantiate a Counterfactual NodeProvider
-2. Connect the NodeProvider to a blockchain contract through an AppFactory instance
-3. Use the AppFactory’s `.proposeInstallVirtual()` method to propose a new virtual state channel based on the AppFactory instance’s settings (including stakes and game logic)
-4. Use the NodeProvider’s `.on()`  method to listen for accepted installs and updated state in the channel
-5. Use the AppInstance’s `.takeAction()` method to propose updates to state in the channel
-6. Use the AppInstance’s `.uninstall()` method to propose closing and resolving the channel
+# Getting Started
 
 ----------
 
-# HighRoller.js
 
-## Truffle unbox
+## Counterfactual's Truffle box
 
 
-We’ll start our new Counterfactual project with the template in the Counterfactual Truffle box. After installing [Truffle](https://truffleframework.com/tutorials/pet-shop), unbox the Counterfactual Truffle box:
+We’ll start our new Counterfactual project with the template in the Counterfactual Truffle box. After installing [Truffle](https://truffleframework.com/docs/truffle/getting-started/installation), and familiarizing yourself with how to [create a new Truffle project](https://truffleframework.com/docs/truffle/getting-started/creating-a-project), unbox the Counterfactual Truffle box:
 
 ```bash
 mkdir my-project
 cd my-project
 truffle unbox counterfactual/truffle-box
 ```
 
-Looking through the template, you’ll find:
+In the box, you'll find `src/app.js`. In this UI template, you'll find: 
 
 * some initialized variables ( `let web3Provider, nodeProvider;` )
 * the async function `run()` which contains calls to
-    * initWeb3() // initializes web3; already complete and explained inline
-    * initContract() // **this is where we point to the blockchain contract for our state channel** - we’ll explain and fill in the details during the guide
-    * setupCF() // setup for the Counterfactual NodeProvider; already complete
-    * install() // **this is the game** - we’ll explain and fill this in
-        * The install function will call the rest of the functions in the doc
+    * initWeb3() // initializes web3; no need to touch this
+    * initContract() // use Truffle to create a Contract object, which enables dynamic fetching of the contract address and content
+    * setupCF() // setup for the Counterfactual NodeProvider; no need to touch this
+    * install() // **this is where we'll code the application UI**
 * a call to the `run()` function
 
+Throughout this guide, we'll be using this template as a reference for developing `app.js`.
+
 ----------
 
 ## Constants
 
-We’ll import some [ethers.js](https://docs.ethers.io/ethers.js) [constants and utilities](https://docs.ethers.io/ethers.js/html/api-utils.html) we’ll need to write High Roller:
+We’ll import some [ethers.js](https://docs.ethers.io/ethers.js) [constants and utilities](https://docs.ethers.io/ethers.js/html/api-utils.html) we’ll need to write **HighRoller**:
```

Comment:
We should introduce what `ethers.js` is. A one-liner could be:

"We're going to use a common ethereum utility library called ethers.js for some parts of this app."